### PR TITLE
Implement Buffer.byteLength

### PR DESCRIPTION
**Pull request by chunseoklee**
IoT.js-DCO-1.0-Signed-off-by: Chunseok Lee chunseok.lee@samsung.com

### CODE REVIEW

File: src/js/buffer.js [hunk 1]
```diff
@@ -48,9 +48,29 @@ function Buffer(subject) {
 
 
 // Buffer.byteLength(string)
-Buffer.byteLength = function(str) {
-  // FIXME: Returns actual byte length of string not counts of characters.
-  return str.length;
+Buffer.byteLength = function(str, enc) {
```
Comment (ILyoan) [inline src/js/buffer.js]: As we are currently supporting only 'utf8' for string encoding. that I think we don't need to get additional parameter for distinguishing string encoding. Furthermore other than 'utf8' encoding should cause raising of an exception like `Error: unsupported encoding`.

**Comment by chunseoklee** [comment]
@ILyoan updated as commented. 
utf8 is on default unless specific encoding option is given.

**Comment by chunseoklee** [comment]
will be revised after this issue [https://github.com/Samsung/jerryscript/issues/722] is clear.

**Comment by chunseoklee** [comment]
This patch suspended due to low priority

**Pull request closed without merging**